web/pages/profile.tsx

```diff
@@ -0,0 +1,716 @@
+import {
+  Button,
+  Flex,
+  Stack,
+  Text,
+  IconButton,
+  Input,
+  Box,
+} from "@chakra-ui/react";
+
+import { EditIcon } from "@chakra-ui/icons";
+
+import React from "react";
+import Select from "react-select";
+
+export default function Profile() {
+  const [editing, setEditing] = React.useState(false);
+
+  const options = [
```

Comment:
Consider giving these options (options, options1, options2, etc.) more descriptive names instead of numerical values.

Reply:
For the value of these dropdowns, use enums instead of string literals, see `utils/types/post.ts`. Note that the breeds don't have enums yet so you might want to create those.